plugins/exec-cronjob.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: exec-cronjob
+spec:
+  version: v0.4.1
+  platforms:
+  - selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    uri: https://github.com/thecloudnatives/kubectl-plugins/archive/v0.4.1.zip
+    sha256: fc5410ebe09295a75ea9a67dd10ba83b3c6e5043ea9f0cf98897e362c665003a
+    files:
+    - from: kubectl-plugins-*/exec-cronjob/*
+      to: "."
+    bin: kubectl-exec_cronjob.sh
+  shortDescription: Run a cron job immediately
+  description: |
+    Run a cron job immediately by extracting the Job spec and creating a Job
```

Comment:
I think this can be just `Run a CronJob immediately as Job` as most people would understand what it does.

Similarly, let's preserve Kubernetes API casing like CronJob.